pss/trojan/message.go

```diff
@@ -217,4 +217,4 @@ func (m *Message) UnmarshalBinary(data []byte) (err error) {
 	m.payload = data[34:payloadEnd]
 	m.padding = data[payloadEnd:]
 	return nil
-}
+}
```

Comment:
add a newline here to fix the linter error

Reply:
fixed